main.go

```diff
@@ -380,6 +383,21 @@ func main() {
 		}()
 	}
 
+	if err := handleLokiLogPipeline(context.Background(), k8sClient); err != nil {
+		setupLog.Error(err, "Failed to handle Loki LogPipeline")
+		os.Exit(1)
+	}
+
+	// Ensure reconciliation of the optional Loki LogPipeline
+	go func() {
+		for range time.Tick(1 * time.Minute) {
+			if err := handleLokiLogPipeline(context.Background(), k8sClient); err != nil {
+				setupLog.Error(err, "Failed to handle Loki LogPipeline")
+				os.Exit(1)
```

Comment:
We should not quit here. An error could easily happen here, for instance if the API server is not reachable. Just log the error and retry in the next iteration.

Reply:
yes, I agree  👍🏻